tests/build/jvm-build.go

```diff
@@ -60,68 +86,84 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jv
 						klog.Infof("error getting logs for pod/container %s/%s: %s", pod.Name, c.Name, cerr.Error())
 						continue
 					}
-					podLog = fmt.Sprintf("%s\n%s\n", podLog, cLog)
+					filename := fmt.Sprintf("%s-pod-%s-%s.log", pod.Namespace, pod.Name, c.Name)
+					toDebug[filename] = cLog
 				}
-				klog.Info(podLog)
 			}
 			// let's make sure and print the pr that starts the analysis first
-			logs, err := f.TektonController.GetPipelineRunLogs(prGeneratedName, testNamespace)
+
+			logs, err := f.TektonController.GetPipelineRunLogs(componentPipelineRun.Name, testNamespace)
 			if err != nil {
 				klog.Infof("got error fetching PR logs: %s", err.Error())
 			}
-			klog.Infof("failed PR logs: %s", logs)
+			filename := fmt.Sprintf("%s-pr-%s.log", testNamespace, componentPipelineRun.Name)
+			toDebug[filename] = logs
+
 			prList, err := f.TektonController.ListAllPipelineRuns(testNamespace)
 			if err != nil {
 				klog.Infof("got error fetching PR list: %s", err.Error())
 			}
 			klog.Infof("total number of pipeline runs not pruned: %d", len(prList.Items))
 			for _, pr := range prList.Items {
-				if pr.Name == prGeneratedName {
+				if pr.Name == componentPipelineRun.Name {
 					continue
 				}
 				prLog, err := f.TektonController.GetPipelineRunLogs(pr.Name, pr.Namespace)
 				if err != nil {
 					klog.Infof("got error fetching PR logs for %s: %s", pr.Name, err.Error())
 				}
-				klog.Infof("pipeline run log for %s: %s", pr.Name, prLog)
+				filename := fmt.Sprintf("%s-pr-%s.log", pr.Namespace, pr.Name)
```

Comment:
yeah I suspect until we can change the pruning we won't get much here, but let's leave it here for when we can make those pruning changes